task4/utils/utils.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+func DateParse(dateStr string) time.Time {
+	date, err := time.Parse("2006-Jan-02", dateStr)
```

Comment:
Usually you'll want to pop error up using `errors.Wrap` or similar one. But for now, it's OK.